gen/build_deploy/bash/dcos_generate_config.sh.in

```diff
@@ -50,4 +55,16 @@ else
     trap 'docker kill {genconf_tar}' HUP QUIT INT TERM
     docker run --rm --name=$DCOS_INSTALLER_CONTAINER_NAME -i -p $PORT:9000 -v $(pwd)/genconf/:/genconf {docker_image_name} "$@"
 fi
-exit $?
+
+if [ -d genconf/ca ]; then
+    psk=$(cat genconf/ca/psk 2>/dev/null || echo "")
+    if [ -z "$psk" ]; then
+        echo "WARNING: DC/OS Bootstrap CA service PSK is unset"
+    fi
+
+    echo "Starting DC/OS Bootstrap CA service in the background"
+    genconf/bin/$DCOS_BOOTSTRAP_CA -d genconf/ca/ serve --address :443 --psk "$psk" &
+    echo
+fi
+
+exit 0
```

Comment:
Before this patch we had `exit $?`. Now we have `exit 0`. What's the rationale? Does not need to be answered before merging.

Reply:
Since the script is running with `set -e` it only gets here if the exit code is 0